src/Page/Guide/Data.elm

```diff
@@ -19,6 +19,7 @@ type alias Guide =
     , slug : String
     , fullTextMarkdown : String
     , summary : String
+    , maybeFeatured : Maybe String
```

Comment:
Should probably be a `Bool` if it's a flag

Reply:
That makes more sense but then we'd need to parse the JSON into a bool, which i am not familiar with